models/profile.js

```diff
@@ -0,0 +1,12 @@
+
+const mongoose = require("mongoose")
+
+const memberSchema = mongoose.Schema({
+    nickname: String,
+    name: String,
+    picture: String,
+    updated_at: String,
+    sub: String
+})
+
+export default mongoose.models.profile || mongoose.model("profile", memberSchema);
```

Comment:
For clarification, what's the difference between `profile.js` and `fellow.js`?

Reply:
fellow.js is a cache of the fellow from GitHub and profile.js are all the user logged in a system

Reply:
Noted, thanks.